actors/common/src/main/java/org/sunbird/learner/util/ContentSearchUtil.java

```diff
@@ -52,6 +52,27 @@
         Unirest.post(contentSearchURL).headers(getUpdatedHeaders(headers)).body(body);
     Future<HttpResponse<JsonNode>> response = RestUtil.executeAsync(request);
 
+    return commonResponseMap(response);
+  }
+
+  public static Future<Map<String, Object>> searchContent(
```

Comment:
Also, searchContent(3 arguments) move all code to that function, handling the optional argument which is query string. 
searchContent(2 arguments) - will invoke searchContent(3 arguments) by passing optional param as null. 

This will help in reducing the code-duplicacy